code/mission/missionmessage.cpp

```diff
@@ -294,6 +294,19 @@ void persona_parse()
 			WarningEx(LOCATION, "Unknown species in messages.tbl -- %s\n", cstrtemp );
 	}
 
+	if (optional_string("$Allow substitution of missing messages:")) {
+		bool Allow_substitution;
```

Comment:
Minor style issue, normally we use lower case at the beginning of local variables.